test/testdata/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos:latest
+FROM registry.centos.org/centos/centos:latest
```

Comment:
To save space, we can use a smaller image like alpine linux..

Reply:
it is about a consistent scenario. images like these can be 'cached' and tagged beforehand so space should not be an issue